client/src/interface/Profile.ts

```diff
@@ -9,3 +9,13 @@ export interface Profile {
   address: string;
   description: string;
 }
+
+export interface ProfileLists {
```

Comment:
```suggestion
export interface SitterProfile {
```

I don't think this is a list of profiles